template1/style.css

```diff
@@ -6,6 +6,11 @@ h1 {
   color: navy;
   margin-left: 20px;
 }
- div{
-   border: 1px solid black;
- }
+.container {
+  display: flex;
+  ...
+  gap: 10px;
+  gap: 10px 20px; /* row-gap column gap */
+  row-gap: 10px;
+  column-gap: 20px;
```

Comment:
What does this do? What were you trying to accomplish. I've never seen these properties before, but they look like they're related to tables, which index.html does not use.